src/App.vue

```diff
@@ -16,32 +16,40 @@
                         <a
                             href="#"
                             aria-current="page"
-                        >Chooser</a>
+                        >{{ $t('app.page-title') }}</a>
                     </li>
                 </ul>
             </nav>
-            <div class="page-head">
-                <div class="select-license-column">
-                    <h2 class="vocab">
-                        {{ $t('select-license.heading') }}
-                    </h2>
-                    <p class="stepper-instructions vocab-body body-bigger">
-                        {{ $t('select-license.instructions') }}
-                    </p>
+
+            <h2>
+                {{ $t('chooser.heading') }}
+            </h2>
+            <p class="stepper-instructions body-bigger">
+                {{ $t('chooser.instructions') }}
+            </p>
+
+            <div class="columns wider-gap">
+                <div class="column">
+                    <Stepper
+                        v-model="currentStepId"
+                        @restart="restart"
+                        @done="done"
+                    />
+                    <help-section />
                 </div>
-                <LocaleChooser class="locale-chooser" />
-            </div>
-            <div class="columns">
-                <Stepper v-model="currentStepId" />
                 <div class="column">
-                    <div class="fixed-right-column">
-                        <LicenseDetailsCard
-                            v-if="showLicense"
-                        />
+                    <div :class="{ 'fixed-right-column': currentStepId !== 7 }">
```

Comment:
Since we are comparing `currentStepId` to 7 several times, would be a good idea to make it a computed property and give it a name, because why step 7 is special is not clear to me.

Reply:
Normally, the right column is fixed to make sure that the License details card is visible on the screen no matter which step is active. When we come to the 7th step (with the Attribution Details form), we show the 'Mark your license' card and the right column should not be fixed so that the 'Mark your license' card is scrolled into view.

Reply:
Please put this as a comment on the function itself.

Reply:
Sure!